src/index.js

```diff
@@ -1,14 +1,100 @@
-import _ from 'lodash';
-import './style.css';
-
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component())
\ No newline at end of file
+import './style.css';
+
+// Lodash, now imported by this script
+
+const list = document.getElementById('uList');
+// const entryTask = document.getElementById('newTask');
```

Comment:
- [ ] Please remove all the commented code from your files as it makes your code look unorganised, messy, and cluttered.